packages/lit-dev-content/site/tutorials/index.html

```diff
@@ -11,22 +11,18 @@
 {% endblock %}
 
 {% block content %}
-<section class="tutorial-category">
-  <h1 class="category-title" id="learn-section">Learn</h1>
-  <div class="category-subtitle">Learn the features of Lit!</div>
+<section class="tutorial-category featured">
+  <h1 class="category-title" id="learn-section">Featured Tutorials</h1>
+  <div class="category-subtitle"></div>
   <div class="center">
     <div class="card-grid">
-      {% for tutorial in tutorials %}
-        {% if tutorial.category == 'Learn' %}
+      {% for tutorial in tutorials | sort(false, false, 'featured.position') %}
+        {% if tutorial.featured.position !== undefined and tutorial.featured.position < 3 %}
           <litdev-tutorial-card
-              size="{{ tutorial.size }}"
+              size="small"
```

Comment:
What happens if there is too much content - I.e. the featured tutorial was originally on a larger card?

Reply:
I was thinking about this. The real thing would be to add "featured" description, but that's way too much work for the author. In the meantime, if it's too long, then it won't be featured. This is an issue we can tackle down the line